common/environment/environment.cpp

```diff
@@ -1668,22 +1666,23 @@ IConstDropZoneInfo * CLocalEnvironment::getDropZoneByAddressPath(const char * ne
                     LOG(MCdebugInfo, unknownJob, "Drop zone coputer: '%s', IP: '%s'", dropZoneMachineName.str(), dropZoneMachineNetAddressIpString.str());
 #endif
                     if (targetIp.ipequals(dropZoneMachineIp))
-                       break;
-                    else
-                       // Path matched, but machine didn't
-                       dropZone = nullptr;
-                }
+                    {
+                        // Keep this drop zone
+                       if (dropzonePathLen > fullDropZoneDir.length())
```

Comment:
I think if the dropzonePathLen is equal to fullDropZoneDir.length() then it means (to me) we already have that path. I don't think it can be add any further information to store same value into the variables in the conditional block. (Perhaps same drop zone name can't appear in different DropZone defininition, but I'm not sure.)
Why do you think it would be useful?